src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.ts

```diff
@@ -46,4 +51,8 @@ export class UpgradeComponent implements OnInit {
     this.healthData$ = this.healthService.getMinimalHealth();
     this.fsid$ = this.healthService.getClusterFsid();
   }
+
+  startUpgradeModal() {
+    this.modalRef = this.modalService.show(UpgradeStartModalComponent);
```

Comment:
```suggestion
    this.modalRef = this.modalService.show(UpgradeStartModalComponent, {
        versions: this.upgradableVersions
    });
```

you have to get the `upgradableVersions` using a `pipe` from above where the `upgradeInfo$` is instanitated.

```
this.upgradeInfo$ = this.upgradeService.list().pipe(
      tap((upgradeInfo: UpgradeInfoInterface) => this.upgradableVersions = upgradeInfo.versions));
 
 ```